provisionning-files/external-provider/external-provider4.8.0-jahia8.2.0.7.yml

```diff
@@ -0,0 +1,43 @@
+- addMavenRepository: 'https://store.jahia.com/nexus/content/repositories/jahia-public-app-store@id=JahiaStore'
+- addMavenRepository: 'https://devtools.jahia.com/nexus/content/groups/public/@snapshots@id=JahiaSnapshot'
+- addMavenRepository: 'https://devtools.jahia.com/nexus/content/repositories/jahia_staging_releases-2772'
+  username: "NEXUS_USERNAME"
+  password: "NEXUS_PASSWORD"
+
+# This modules were added during the switch from using jahia-discovery to jahia-ee
+- installBundle:
+  - 'mvn:org.jahia.modules/legacy-default-components'
+  - 'mvn:org.jahia.modules/press'
+  - 'mvn:org.jahia.modules/person'
+  - 'mvn:org.jahia.modules/news'
+  - 'mvn:org.jahia.modules/channels'
+  - 'mvn:org.jahia.modules/font-awesome/6.1.5'
+  - 'mvn:org.jahia.modules/calendar'
+  - 'mvn:org.jahia.modules/bootstrap3-core'
+  - 'mvn:org.jahia.modules/bootstrap3-components'
+  - 'mvn:org.jahia.modules/location'
+  - 'mvn:org.jahia.modules/topstories'
+  - 'mvn:org.jahia.modules/rating'
+  - 'mvn:org.jahia.modules/event'
+  - 'mvn:org.jahia.modules/bookmarks'
+  - 'mvn:org.jahia.modules/dx-base-demo-core'
+  - 'mvn:org.jahia.modules/dx-base-demo-templates'
+  - 'mvn:org.jahia.modules/dx-base-demo-components'
+  - 'mvn:org.jahia.modules/digitall/3.0.0' 
+  - 'mvn:org.jahia.modules/skins'
+  - 'mvn:org.jahia.modules/default-skins'
+  - 'mvn:org.jahia.modules/grid'
+  - 'mvn:org.jahia.modules/tabularList'  
+  autoStart: true
+  uninstallPreviousVersion: true
+
+- import: "jar:mvn:org.jahia.modules/digitall/3.0.0/zip/import!/users.zip"
+- importSite: "jar:mvn:org.jahia.modules/digitall/3.0.0/zip/import!/Digitall.zip"
+
+- installBundle:
+  - 'mvn:org.jahia.test/jahia-test-module/4.8.0'
```

Comment:
should be a different version for jahia-test-module ? maybe 8.2.0.7 based on filename.